src/vos/tests/evt_ctl.sh

```diff
@@ -9,6 +9,14 @@ DAOS_DIR=$(cd "${cwd}/../../.." && echo "$PWD")
 #shellcheck disable=SC1090
 source "$DAOS_DIR/.build_vars.sh"
 EVT_CTL="$SL_PREFIX/bin/evt_ctl"
+if [ -d "$DAOS_DIR/test_results" ]
+then
+    rm "$DAOS_DIR"/test_results/evt*.xml
+else
+    mkdir "$DAOS_DIR/test_results"
+fi
+CMOCKA_MESSAGE_OUTPUT="xml"
```

Comment:
(lint) CMOCKA_MESSAGE_OUTPUT appears unused. Verify it or export it. [SC2034]

Reply:
The linter tool is correct (shellcheck is what is used if you want to run it locally).   You need to export it to get the value in the child or you need to place it on same command line.